username_api.py

```diff
@@ -43,6 +43,12 @@ def get_avatar(website, username):
         if not result or not result[0]:
             return None
         result = result[0]
+    elif 'html_selector' in data:
+        page = r.get(get_profile_url(website, username))
+        soup = BeautifulSoup(page.content, 'html.parser')
```

Comment:
don't need to fetch page content again. it's already fetched at: https://github.com/manu-chroma/username-availability-checker/blob/c29956f7a2256113bd134b36de6dd452634de732/username_api.py#L36

Reply:
That made it faster. Thanks